src/libraries/Microsoft.VisualBasic.Core/Directory.Build.props

```
@@ -3,8 +3,8 @@
   <PropertyGroup>
     <!-- We need to set MajorVersion so that FileVersion is set correctly and is
     greater than the one in the previous release -->
-    <MajorVersion>$([MSBuild]::Add($(MajorVersion), 5))</MajorVersion>
-    <AssemblyVersion>$(MajorVersion).$(MinorVersion).0.0</AssemblyVersion>
+    <MajorVersion>$([MSBuild]::Add($(PreviousReleaseMajorVersion), 5))</MajorVersion>
+    <AssemblyVersion>$(PreviousReleaseMajorVersion).$(MinorVersion).0.0</AssemblyVersion>
```

This shouldn't be changed as the version is automatically calculated based on the Versions.props MajorVersion property.

```suggestion
    <MajorVersion>$([MSBuild]::Add($(MajorVersion), 5))</MajorVersion>
    <AssemblyVersion>$(MajorVersion).$(MinorVersion).0.0</AssemblyVersion>
```